main.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/242e63f88a.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round:ital@0;1&family=Inter:wght@100;300;400;500;700;900&family=Poppins:wght@100;400;500;700;900&family=Roboto:wght@100;400;500;700;900&display=swap" rel="stylesheet">    
+    <title>Portfolio</title>
+</head>
+<body>
+    <section id="main-section">
+        <div id="bar-container">
+            <ul id="status-bar">
+                <li class="time">9:41</li>
+                <li class="fas-container">
+                    <i class="fas fa-signal"></i>
+                    <i class="fas fa-wifi"></i>
+                    <i class="fas fa-battery-full"></i>
+                </li>
+            </ul>
+            <header id="tool-bar">
+                <a class="logo" href="#">John Doe</a>
+                <i class="fas fa-bars"></i>
```

Comment:
Kindly make the hamburger button `<i class="fas fa-bars"></i>`, inside `nav` tag.
Like  `<nav><i class="fas fa-bars"></i></nav>`.